### PR TITLE
update team org chart

### PR DESCRIPTION
**Pull request by sashaostrikov**

### CODE REVIEW

File: data/product_teams.yml [hunk 1]
```diff
@@ -65,7 +65,6 @@ iam:
   brief: The IAM team is responsible for both authentication and authorization to Sourcegraph, including login and user management for on-prem and managed instances.
   strategy_link: /strategy-goals/strategy/iam/index.md
   pm: ryan_phillips
-  em: jordan_plahn_iam
```
Comment (indradhanush) [inline data/product_teams.yml:68]: Should add that Diego is interim.

File: data/product_teams.yml [hunk 2]
```diff
@@ -92,7 +91,6 @@ repo_management:
   title: Repo Management
   strategy_link: /strategy-goals/strategy/repo-management
   pm: ryan_phillips
-  em: jordan_plahn
```
Comment (indradhanush) [inline data/product_teams.yml:95]: Same.

**Review by indradhanush — CHANGES_REQUESTED** [review]

File: data/product_teams.yml (hunk 1 above)
Comment (sashaostrikov) [reply data/product_teams.yml:68]: fixed

File: data/product_teams.yml (hunk 2 above)
Comment (sashaostrikov) [reply data/product_teams.yml:95]: fixed

**Review by indradhanush — APPROVED** [review]

**Pull request merged**